phpcs.xml.dist

```diff
@@ -37,6 +37,9 @@
 	</rule>
 
 	<rule ref="WordPress">
+		<exclude name="Generic.CodeAnalysis.AssignmentInCondition.Found"/>
```

Comment:
Do we really want to exclude this kind of sniff?
Adding it requires to many changes?

Reply:
No I don't want to exclude it in the long term. Yes, it's a lot of work to follow this rule. Especially in Polylang as it's the oldest plugin and I used this a lot.